Server/src/controllers/review.controller.js

```diff
@@ -0,0 +1,88 @@
+import mentors from '../models/mentor.model';
+import users from '../models/users.model';
+import session from '../models/mentorship.model';
+import review from '../models/review.model';
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+
+class ReviewController {
+  /**
+  * Review mentor after mentorship session
+  * @param {object} req
+  * @param {object} res
+  */
+  static reviewMentor(req, res) {
+    const isMentorshipSession = session.find(s => s.sessionId == parseInt(req.params.sessionId));
```

Comment:
Make use of object destructuring for this

` const { sessionId } = req.params `